pdk/1.0/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

```diff
@@ -271,6 +294,52 @@ properties:
       default: {"id": "1", "name": "John Doe"}
 ----
 
+[[dataweave-expressions]]
+=== DataWeave Expressions
+
+The `dataweave` format for `string` type parameters specifies the input parameter as a DataWeave expression.
+
+The `dataweave` variable supports the following expressions bindings:
+
+[source,yaml]
+----
+<configuration-parameter>:
+ type: string
+ format: dataweave
+ bindings:
+   payload:
+     mimeType: text
+   attributes: true
+   authentication: false
+   vars: []
+----
+
+* `payload`: The body of the imcoming request. Define the `mimeType` as either `text` or `json` to set how to read the payload. By default, `mimeType` is set to `text`. To restrict the use of the `payload` variable, you must exclude the payload variable, for example:
++
+[source,yaml]
+----
+bindings:
+   attributes: true
+   authentication: false
+   vars: []
+----
+
+* `attributes`: Attributes of the incoming request such as headers. Set `attributes` as `true` to enable the policy to read the attributes of the incoming request. By default, attributes is set to `true`.
+* `authentication`: Authentication information of the incoming request. Set `authentication` to `true`to enable the policy to read authenticaton details of the incoming request. By default, `authentication is set to false.
```

Comment:
```suggestion
* `authentication`: Authentication information of the incoming request. Set `authentication` to `true` to enable the policy to read authentication details of the incoming request. By default, `authentication` is set to `false`.
```

Reply:
In these bullets - should "of" be "for" or "in"?

Reply:
I think "of" is correct as the request owns the authentication information.